api4/cloud.go

```diff
@@ -387,6 +387,29 @@ func handleCWSWebhook(c *Context, w http.ResponseWriter, r *http.Request) {
 			c.Err = nErr
 			return
 		}
+	case model.EventTypeSendFirstCloudAdminWelcomeEmail:
+		user, err := c.App.GetUserByUsername(event.CloudWorkspaceOwner.UserName)
+		if err != nil {
+			c.Err = model.NewAppError("Api4.handleCWSWebhook", err.Id, nil, err.Error(), err.StatusCode)
+			return
+		}
+
+		team, err := c.App.GetTeamByName(event.CloudWorkspaceOwner.FirstTeamName)
```

Comment:
Maybe here we can just get teams and take the first one, at the time this webhook is called there is only one team created.